### PR TITLE
Optionally adds species URL to download #945

### PR DESCRIPTION
**Pull request by chrisala**

### CODE REVIEW

File: src/main/groovy/au/org/ala/ecodata/metadata/SpeciesUrlGetter.groovy [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    @Override
+    def species(Object node, Value outputValue) {
+        def val = outputValue.value
+        if (!val?.name) {
+            return ""
+        }
+
+        return val?.guid ? biePrefix+val.guid : "Unmatched name"
```
Comment (temi) [inline src/main/groovy/au/org/ala/ecodata/metadata/SpeciesUrlGetter.groovy:19]: should there be a check for "A_GUID"?

File: grails-app/conf/application.groovy [hunk 2]
```diff
@@ -501,6 +501,9 @@ app {
 if (!ala.baseURL) {
     ala.baseURL = "https://www.ala.org.au"
 }
+bie.ws.url = "https://www.bie.ala.org.au/ws"
```
Comment (temi) [inline grails-app/conf/application.groovy:504]: is the wrong address done intentionally?

File: src/main/groovy/au/org/ala/ecodata/metadata/SpeciesUrlGetter.groovy (hunk 1 above)
Comment (chrisala) [reply src/main/groovy/au/org/ala/ecodata/metadata/SpeciesUrlGetter.groovy]: Thanks, yes there should be - fixed.

File: grails-app/conf/application.groovy (hunk 2 above)
Comment (chrisala) [reply grails-app/conf/application.groovy]: Thanks, fixed.  (I've also added the bie.ws.url to the ansible version of the config file)

**Review by temi — APPROVED** [review]

**Pull request merged**